### ISSUE TITLE
Support for translator comments

### ISSUE THREAD

**Issue by atimmer** [praekelt/jspot#37]
Currently jspot has no support for translator comments (In get text terms: extracted comments). By default the `xgettext` command and this is a must have for translation javascript files.

**Comment by justinvdm** [comment]
Indeed, this is something jspot is lacking. Will try find time to add this, though it might not be very soon.
